templates/menu.ftl

```diff
@@ -49,7 +49,8 @@
                         <li><a href="http://training.nextflow.io">Nextflow training</a></li>
                         <li><a href="http://nextflow-io.github.io/patterns/index.html">Implementation patterns</a></li>
                         <li><a href="https://github.com/nextflow-io/nextflow">GitHub repository</a></li>
-                        <li><a href="https://github.com/nextflow-io/nextflow/discussions">Discussion forum</a></li>
+                        <li><a href="https://community.seqera.io">Community forum</a></li>
```

Comment:
Should maybe link directly to the Nextflow category..

```suggestion
                        <li><a href="https://community.seqera.io/c/nextflow/5">Community forum</a></li>
```

Reply:
👍